PIPs/pip-27.md

```diff
@@ -0,0 +1,127 @@
+---
+pip: 27
+title: Pruned Node
```

Comment:
I think it's better to call it light node or light client since every protocol did it. I mean everyone knows this concept as the light client so it can cause misunderstanding.